pkg/swiveltesting/dep.go

```diff
@@ -21,6 +21,8 @@ type Dep struct {
 	CompoundTokenAddress common.Address
 	Erc4626              *mocks.Erc4626
 	Erc4626Address       common.Address
+	YearnVault           *mocks.YearnVault
+	YearnVaultAddress    common.Address
```

Comment:
the names are a bit more specific in mocks and tests as they need to not clash with mock adapters. these names don't effect the actual protocol at all, as, they are just here for testing